tools/prlint/package.json

```diff
@@ -9,6 +9,9 @@
     "name": "Amazon Web Services",
     "url": "https://aws.amazon.com"
   },
+  "scripts": {
+    "build": "echo success"
+  },
```

Comment:
Why? This is pointless...

Reply:
so that i can run `scripts/foreach yarn test`.

Happy to accept other solutions... :)

Reply:
I believe `yarn run build` should work (foreach has provisions for that).

Reply:
Also I don't mind this one bit